Makefile

```diff
@@ -53,9 +53,11 @@ else
 endif
 
 export TARGET_OUT ?= $(shell pwd)/out/$(TARGET_OS)_$(TARGET_ARCH)
+export TARGET_OUT_LINUX ?= $(shell pwd)/out/linux_amd64
 
 ifeq ($(BUILD_WITH_CONTAINER),1)
 export TARGET_OUT = /work/out/$(TARGET_OS)_$(TARGET_ARCH)
+export TARGET_OUT_LINUX = /work/out/linux_amd64
```

Comment:
nit: not needed since common files update

Reply:
For whatever reason, master does not have this Makefile change. see https://github.com/istio/istio/blob/master/Makefile#L58.

Reply:
See PR: https://github.com/istio/istio/pull/19672